presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -389,7 +389,7 @@ private static void initializeDeserializer(Configuration configuration, Deserial
             configuration = copy(configuration); // Some SerDes (e.g. Avro) modify passed configuration
             deserializer.initialize(configuration, schema);
         }
-        catch (SerDeException e) {
+        catch (SerDeException | RuntimeException e) {
```

Comment:
I would squash this commit into the one that modifies the next line.

Reply:
I know the changes are rather tiny, but logically they are not quite related, this is why i composed separate commits.